api/views.py

```diff
@@ -138,3 +143,17 @@ def _formatted_bad_request(format_):
     else:
         # json
         return JsonResponse({}, status=400)
+
+@api_view([GET])
+def enrichment_view(request):
+    observable_name = request.query_params.get('query')
+    logger.info(f"Enrichment view requested for: {str(observable_name)}")
+    try:
+        required_object = IOC.objects.get(name=observable_name)
+    except IOC.DoesNotExist:
+        return Response({'found':"false"}, status=status.HTTP_200_OK)
+    serialized = IOCSerializer(required_object)
+    response_data = serialized.data
```

Comment:
- missing `serializer.is_valid()` statement.
- should use `serializer.validated_data` to get data.
- Define also the `found` attribute in the `IoCSerializer`. You can define it as a `BooleanField(read_only=True)`.